v2.0/select-clause.md

```diff
@@ -0,0 +1,451 @@
+| `AS OF SYSTEM TIME timestamp` | Retrieve data as it existed as of [`timestamp`](timestamp.html).<br/><br/>For more information, see [this example](#select-historical-data-time-travel) or our blog post [Time-Travel Queries](https://www.cockroachlabs.com/blog/time-travel-queries-select-witty_subtitle-the_future/).<br/><br/>Because `AS OF SYSTEM TIME` returns historical data, your reads might be stale. |
+| `WHERE a_expr` | Only retrieve rows that return `TRUE` for `a_expr`, which must be a [scalar expression](scalar-expressions.html) that returns Boolean values using columns (e.g., `<column> = <value>`).  |
+| `GROUP BY expr_list` | When using [aggregate functions](functions-and-operators.html#aggregate-functions) in `target_elem` or `HAVING`, list the column groupings in `expr_list`. |
+| `HAVING a_expr` | Only retrieve aggregate function groups that return `TRUE` for `a_expr`, which must be a [scalar expression](scalar-expressions.html) that returns Boolean values using an aggregate function (e.g., `<aggregate function> = <value>`). <br/><br/>`HAVING` works like the `WHERE` clause, but for aggregate functions. |
+| `WINDOW window_definition_list` | A list of [window functions definitions](window-functions.html) |
+
+{{site.data.alerts.callout_info}}Using two or more table expressions in the <code>FROM</code> sub-clause,
+separated with a comma, is equivalent to a <a href="joins.html"><code>CROSS JOIN</code> expression</a>.{{site.data.alerts.end}}
+
+## Eliminate Duplicate Rows
+
+The `DISTINCT` subclause specifies to remove duplicate rows.
+
+By default, or when `ALL` is specified, `SELECT` returns all the rows
+selected, without removing duplicates. When `DISTINCT` is specified,
+duplicate rows are eliminated.
+
+Without `ON`, two rows are considered duplicates if they are equal on
+all the results computed by `SELECT`.
+
+With `ON`, two rows are considered duplicates if they are equal only
+using the [scalar expressions](scalar-expressions.html) listed with `ON`. When two rows are considered duplicates according to `DISTINCT ON`, the values from the first `FROM` row in the order specified by [`ORDER BY`](query-order.html) are used to compute the remaining target expressions. If `ORDER BY` is not specified, CockroachDB will pick any one of the duplicate rows as first row, non-deterministically.
+
+## Examples
+
+### Choose Columns
+
+#### Retrieve Specific Columns
+
+Retrieve specific columns by naming them in a comma-separated list.
+
+~~~ sql
+> SELECT id, name, balance
+FROM accounts;
+~~~
+~~~
++----+-----------------------+---------+
+| id |         name          | balance |
++----+-----------------------+---------+
+|  1 | Bjorn Fairclough      |    1200 |
+|  2 | Bjorn Fairclough      |    2500 |
+|  3 | Arturo Nevin          |     250 |
+[ truncated ]
++----+-----------------------+---------+
+~~~
+
+#### Retrieve All Columns
+
+Retrieve all columns by using `*`.
```

Comment:
End with a colon and place `{% include copy-clipboard.html %}` on line before code block.